mitodl/ol-infrastructure
# config: enable certificate_html_view to enable certificate option in studio (MITxOnline)

# What are the relevant tickets?
Part of debugging https://github.com/mitodl/hq/issues/1630, Specifically https://github.com/mitodl/hq/issues/1630#issuecomment-1640111969 comment.


# Description (What does it do?)
- This would enable a setting `Certificates` in the Studio course settings
- Enabling the `Certificates` would make it easy for us to check if a course has active certificate pages created.
- Would also allow us to `Activate/Deactivate` the certificate pages.
- More details are in https://github.com/mitodl/hq/issues/1630#issuecomment-1640111969

# Screenshots (if appropriate):
**Certificate setting in Studio**
<img width="303" alt="image" src="https://github.com/mitodl/hq/assets/34372316/645af480-9067-46f6-9918-dbfca850f98a">

**Certificate Page in Studio**

<img width="1498" alt="image" src="https://github.com/mitodl/hq/assets/34372316/cf0836c2-bf37-4fc1-9943-10e0345a17d7">

# How can this be tested?
Once deployed, We should see `Certificate` setting in studio course settings. (See screenshot above)

# Additional Context
This is probably just a temporary change to debug a few things in MITxOnline studio, It's possible that we would revert this once our debugging is complete.

If you think, To enable a feature temporarily have another solution than a PR, Please let me know.
